### ISSUE TITLE
Align color guidance with accessibility.digital.gov

### ISSUE THREAD

**Issue by thisisdano** [uswds/uswds#2791]
We should evaluate our current color guidance and make sure it addresses (and links to) the guidance in https://accessibility.digital.gov/visual-design/color-and-contrast/

- Should we make any extra effort to promote AAA contrast levels?
- Is there any research we can do or find to support exceeding AA contrast?
- Make sure we mention (as we do on the accessibility site) avoiding very high contrast environments

**Comment by maya** [comment]
I support linking to https://accessibility.digital.gov/visual-design/color-and-contrast/ as a resource.

I think it's perfectly reasonable to align with WCAG 2.1, which is AA, technically we only need to follow 2.0, but I'd support 2.1 since it's the future. People can always go above and beyond (AAA) if their project warrants it. 

It could be nice if the magic numbers supported those that wanted AAA, e.g. a magic number of 60 or 70 would achieve AAA, so folks have the option.

>Make sure we mention (as we do on the accessibility site) avoiding very high contrast environments
Probably nice to mention this, but wouldn't make it a requirement.

**Comment by maya** [comment]
To clarify "I'd support 2.1 since it's the future," I meant it's the future of what Section 508 will support at some point, though it is a recommendation by WCAG, so it is current.

**Comment by thisisdano** [comment]
We'll need to test and see if the current grades support any reliable AAA contrast — since the grades are base on relative luminance, it's very possible that they do

**Comment by maya** [comment]
Yes, it appears a difference of 50 will achieve AAA large text and a difference of 70 will achieve AAA normal text (tested with the gray family). AAA normal text is the same requirement as AA large text. We could easily update or documentation to add AAA normal support.

**Comment by thisisdano** [comment]
Excellent — that squares with what I was just about to write :)

**Comment by thisisdano** [comment]
@vizionsinmotion Thanks for reaching out to us via email and for prompting this issue. I was wondering if there were any relevant links or research you could point us to around Irlen Syndrome and light sensitivity that we could use to inform our guidance?

**Comment by vizionsinmotion** [comment]
People with Dyslexia, Irlen Syndrome, and/or light sensitivity could be further challenged with the highest contrast combination of black/dark text on white background.  Some are un-diagnosed so they do not even realize or understand why words jumble, invert, float, shimmer, blur, or disappear or why they get searing migraines while reading.  

Tempering the contrast by changing the background from white to a light color (tan, off-white, etc.) is such a simple change that makes a world of difference to those impacted and with virtually no negative effects to those who are not.  Actually, the optimum contrast combination is white/light colored text on black/dark background which is a max contrast option.  Even better is to offer the users a way to set the color to what best suits them.  Hopefully more people will approach accessibility as a way to lessen the challenges instead of a "nice to have if or when we can get to it"?

https://accessibility.digital.gov/visual-design/color-and-contrast
https://www.w3.org/WAI/RD/2012/text-customization/r11
https://irlen.com/what-is-irlen-syndrome/
https://w3c.github.io/wcag/coga/gap-analysis.html 
https://www.dyslexic.com/blog/quick-guide-making-content-accessible/

Hope this helps!

**Comment by vizionsinmotion** [comment]
@thisisdano - glad you found the info helpful and can appreciate its impacts.  Yes, the irlen.com site defaults to a white background but users can select from different background colors (aka colored glasses) at the top of the page but it is not very obvious.  The dyslexic site is also black text on white despite the content - obvious gap between content vs design.  Even some of the government sites have missed this one also.

Ideally, everyone works together to maximize accessibility - content owners, solution architects, designers, developers, users, etc.  Hopefully this team helps to close the gap!

**Comment by vizionsinmotion** [comment]
A few more related references:

https://www.w3.org/WAI/test-evaluate/preliminary/#contrast
https://www.w3.org/TR/low-vision-needs/#brightness-and-color
https://www.w3.org/TR/low-vision-needs/#text-contrast

**Comment by thisisdano** [comment]
Fixed in https://github.com/uswds/uswds-site/pull/664
